.github/workflows/build.yml

```diff
@@ -190,6 +206,6 @@ jobs:
 
       - name: Retag and push the image
         run: |
-          docker pull "$DOCKER_APP_IMAGE"
-          echo "$DOCKER_METADATA_OUTPUT_TAGS" | tr ' ' '\n' | xargs -n1 docker tag "$DOCKER_APP_IMAGE"
-          docker push --all-tags "$(echo "$DOCKER_APP_IMAGE" | cut -f1 -d:)"
+          docker buildx imagetools create \
+            $(jq -cr '.tags | map("--tag " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
+            $DOCKER_APP_IMAGE_ARM64  $DOCKER_APP_IMAGE_X64
```

Comment:
I don't _think_ there can be special characters or whitespace in those variables, but just in case I would always enclose them in double-quotes. I defer to @awilfox on this however as I think she is more the expert in shell-scripting best practice.

Reply:
fixed in 810bbea